server/controller/src/main/java/ai/starwhale/mlops/datastore/parquet/SwWriteSupport.java

```diff
@@ -212,8 +213,8 @@ private static Pair<BaseValue, Wal.Column.Builder> createWal(ColumnSchema schema
                     if (entry.getValue() != null) {
                         resultValue = SwWriteSupport.createWal(schema.getValueSchema(), entry.getValue());
                     }
-                    if ((resultKey == null || resultKey.getLeft() != null)
-                            && (resultValue == null || resultValue.getLeft() != null)) {
+                    if ((resultKey == null || resultKey.getRight() == null)
```

Comment:
what about simplify this logic as:
`if (resultKey.getRight() == null) && resultValue.getRight() == null)`
`    newMapValue.put(resultKey.getLeft(), resultValue.getLeft()); `
because the result of SwWriteSupport.createWal always have value.